apps/web/src/views/CakeStaking/hooks/useContractWrite/useWriteIncreaseLockWeeksCallback.tsx

```diff
@@ -54,24 +60,36 @@ export const useWriteIncreaseLockWeeksCallback = (onDismiss?: () => void) => {
           setStatus(ApproveAndLockStatus.ERROR)
         }
       }
-    } catch (error) {
+    } catch (error: any) {
       console.error('Failed to increase lock weeks', error)
       if (isUserRejected(error)) {
         setStatus(ApproveAndLockStatus.REJECT)
       } else {
+        logger.warn(
+          'Failed to increase lock weeks',
```

Comment:
Can we add a prefix in front? So we can better filter the log. For example `[CakeStaking]: `